Lib/subprocess.py

```diff
@@ -978,6 +978,21 @@ def __init__(self, args, bufsize=-1, executable=None,
 
             raise
 
+    def __repr__(self):
+        max_args_length = 30
+        args = ' '.join(self.args)
+
+        if len(args) > max_args_length:
+            args = ('%s...' % args[:max_args_length]).strip()
+
+        format_str = "%s.%s: pid:'%s' args:'%s'" % (
+            self.__module__, self.__class__.__name__, self.pid, args)
```

Comment:
I'd drop the module part since it's long enough as it is, and knowing the module isn't that useful.